Formula/crystal.rb

```diff
@@ -50,11 +65,34 @@ def install
     end
 
     ENV["CRYSTAL_CONFIG_PATH"] = prefix/"src:lib"
+    ENV["CRYSTAL_CONFIG_LIBRARY_PATH"] = prefix/"embedded/lib"
     ENV.append_path "PATH", "boot/bin"
 
-    system "make", "deps"
-    (buildpath/".build").mkpath
+    resource("bdw-gc").stage do |stage|
+      system "./configure", "--disable-debug",
+                            "--disable-dependency-tracking",
+                            "--disable-shared",
+                            "--enable-large-config"
+      system "make"
+      system "make", "check"
 
+      @bdwgc_path = Dir.pwd
+      stage.staging.retain!
```

Comment:
Rather than play this whole game with paths, let's just do:

```
resource("bdw-gc").stage buildpath/"gc"
```

then it's in `buildpath/"gc"` and we can use that easily.

Reply:
Sure. I didn't know that. I ended up needing a `cd buildpath/"gc"` to run the `make` withing the stage though.